### PR TITLE
Mark "Mix & OTP" chapters as part of the guide

### PR DESCRIPTION
**Pull request by sotte**
In the beginning of every chapter reference the previous chapter and the first
chapter of the "OTP & Mix" guide.

See #834

I can change the wording if required.

### CODE REVIEW

File: getting-started/mix-otp/agent.markdown [hunk 1]
```diff
@@ -7,6 +7,8 @@ title: Agent
 
 {% include toc.html %}
 
+Note that this chapter is part of the *Mix and OTP* guide and it depends on the previous chapters. Maybe read the [previous chapter](/getting-started/mix-otp/introduction-to-mix.html).
```
Comment (whatyouhide) [inline getting-started/mix-otp/agent.markdown]: I would say 

> Note that this chapter is part of the *Mix and OTP* guide and it depends on previous chapters in this guide. Here's [the previous chapter](...)

Wdyt?

Comment (sotte) [inline getting-started/mix-otp/agent.markdown]: Fine with me!

I'll wait for more feedback and update the PR later.

**Comment by josevalim** [comment]
I would prefer to go with something like:

> This chapter is part of the *Mix and OTP* guide and it depends on previous chapters in this guide. For more information, [read the guide introduction](...) or check out the chapter index in the sidebar.

If we keep it generic, we can even put it in a include and use it in all chapters.

Thoughts?

**Comment by whatyouhide** [comment]
@josevalim yeah, I was leaning on that too tbh. Let's go with a generic link to the whole guide 👍

**Comment by sotte** [comment]
I'm using an `include` now. Didn't touch the intro.

**Comment by josevalim** [comment]
:heart: :green_heart: :blue_heart: :yellow_heart: :purple_heart:

**Pull request merged**